core/primitives/src/sharding/shard_chunk_header_inner.rs

```diff
@@ -7,12 +7,17 @@ use near_primitives_core::hash::CryptoHash;
 use near_primitives_core::types::{Balance, BlockHeight, Gas, ShardId};
 use near_schema_checker_lib::ProtocolSchema;
 
+// When removing CryptoHash fields from new versions to be safe we return defaults instead of
+// panicking.
+const DEFAULT_CRYPTO_HASH: &CryptoHash = &CryptoHash::new();
```

Comment:
nit: CryptoHash::default() is what we usually use. Also, confirming, this is because we can't have a temp variable to return reference?